src/main/scala/mesosphere/mesos/NoOfferMatchReason.scala

```diff
@@ -14,6 +14,7 @@ object NoOfferMatchReason {
   case object UnfulfilledConstraint extends NoOfferMatchReason
   case object AgentMaintenance extends NoOfferMatchReason
   case object NoCorrespondingReservationFound extends NoOfferMatchReason
+  case object DeclinedScarceResources extends NoOfferMatchReason
```

Comment:
you have to add this to the reasonfunnel otherwise it won't appear in the API. I've been there and it sucks and is easy to miss...

Maybe it would be nice to add a test that verifies that it appears in API, then we would see this...